db_schema/schema.sql

```diff
@@ -529,6 +529,13 @@ INSERT INTO appleCountries (id, name, code, ccc) VALUES (4, "Afghanistan", "AF",
 (894, "Zambia", "ZM", "zmb"),
 (716, "Zimbabwe", "ZW", "zwe");
 
+CREATE TABLE IF NOT EXISTS anchorEpisodesPage (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  web_episode_id VARCHAR(128) NOT NULL,
+  PRIMARY KEY (account_id, episode_id, web_episode_id)
```

Comment:
Should the `account_id` be part of the primary key here?

Reply:
yes. I would add an index INDEX (account_id, web_episode_id, episode_id)

Reply:
good idea. Added below